pyproject.toml

```diff
@@ -52,3 +53,7 @@ Changelog = "https://github.com/TUM-DAML/seml/releases/latest"
 
 [project.scripts]
 seml = "seml.__main__:main"
+
+[tool.pyright]
+# TODO: Bump this to "strict" once we have type hints for everything.
```

Comment:
Should this remain a TODO?

Reply:
I don't know; do we ever want proper type checking? Tbh, it seems rather difficult.